subnet/etcdv2/registry.go

```diff
@@ -48,13 +48,14 @@ type Registry interface {
 }
 
 type EtcdConfig struct {
-	Endpoints []string
-	Keyfile   string
-	Certfile  string
-	CAFile    string
-	Prefix    string
-	Username  string
-	Password  string
+	Endpoints      []string
+	Keyfile        string
+	Certfile       string
+	CAFile         string
+	Prefix         string
+	Username       string
+	Password       string
+	PreviousSubnet ip.IP4Net
```

Comment:
Since I was adding a variable to the local manager, I thought that since it was currently being built according to the values passed in from this `EtcdConfig` that this would be the best place to pass in that value.  Looking back at it, this config really only applies to values used to set up a connection to etcd so the value should be moved out for organizational reasons.  Downside of moving it out is it clutters up the `NewLocalManager` constructor a little.